src/status_im/events.cljs

```diff
@@ -1,7 +1,9 @@
 (ns status-im.events
-  (:require [re-frame.core :as re-frame]
+  (:require [clojure.string :as string]
```

Comment:
order namespaces alphabetically please to minimize conflicts with current and future PRs